ovon/task/sensors.py

```diff
@@ -42,7 +42,8 @@ def __init__(
         config: "DictConfig",
         **kwargs: Any,
     ):
-        self.cache = load_pickle(config.cache)
+        # self.cache = load_pickle(config.cache)
+        self.cache = load_pickle('data/text_embeddings/siglip.pkl')
```

Comment:
Why is this hardcoded ? I think this is also overridden in the slurm job script

Reply:
Also you set `habitat_baselines.load_resume_state_config=False` in the slurm batch script so not really sure why this is hardcoded

Reply:
This can be overwritten using slurm job script.
Was using training config earlier which was looking at a different location. Hence had to hardcode it then

Reply:
So if you are anyway overriding with slurm script then remove the hardcoding here otherwise future experiments will silently just use the .pkl at that hardcoded location

Reply:
Outdated